src/utils/referral.js

```diff
@@ -128,13 +128,21 @@ const extractReasonCode = (data) => {
   return [{coding: coding}];
 };
 
-const status = [
+const FHIRServiceRequestStatus = [
   `draft`,
-  `active`,
-  `revoked`,
-  `completed`
+  `revoked`
 ];
 
+const followUpInstruction = {
+  revoked: `Missed visit`,
+  completed: `completed`
+};
+
+const healthFacilityContact = {
+  revoked: `revoked`,
```

Comment:
I'm curious what these states mean with respect to a health facility contact

Reply:
I believe `revoked` means whether the instruction still applies while `completed` is whether it was, well: completed.